datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -65,17 +75,36 @@ def is_metric_in_metadata_file(metric, check):
     return False
 
 
+def has_static_field_changed(manifest_data, check_name, field_to_check, repo_url):
+    repo_url = urljoin(repo_url, f'{check_name}/manifest.json')
+    if MANIFESTS_FROM_MASTER.get(check_name):
+        data = MANIFESTS_FROM_MASTER.get(check_name)
+    else:
+        data = requests.get(repo_url.format(check_name))
+        MANIFESTS_FROM_MASTER[check_name] = data
+
+    if data.status_code == 404:
+        # This integration isn't on the provided repo yet, so it's field can change
+        return False, ""
```

Comment:
Return `None` instead of boolean and string?

Reply:
Good call